CHANGES.md

```diff
@@ -8,6 +8,8 @@ releases are available on [Anaconda.org](https://anaconda.org/conda-forge/gettsi
 
 - {gh}`638` Don't use functions in `compute_taxes_and_transfers` that are not active
   ({ghuser}`lars-reimann`).
+- {gh}`630` Fixes bug in `piecewise_polynomial` that ignored jumps in intercepts
```

Comment:
Should always go on top, timing of commit to main matters, not id.

Reply:
Thanks, I was unsure about this.